torch/quantization/fx/qconfig_utils.py

```diff
@@ -50,12 +50,13 @@ def _convert_to_ordered_dict(key, qconfig_dict):
     _convert_to_ordered_dict('module_name_regex', qconfig_dict)
     _convert_to_ordered_dict('module_name', qconfig_dict)
 
-def get_module_type_qconfig(qconfig_dict, module_type, fallback_qconfig):
+def get_object_type_qconfig(qconfig_dict, object_type, fallback_qconfig):
+    # object_type can be
+    # 1. Module type (call_module)
+    # 2. function (call_function)
+    # 3. string (call_method)
```

Comment:
optional: can we type hint it, something like `Union[torch.nn.Module, Callable, str]`?

Reply:
it's module type not module instance, so I'll use Union[Callable, str] here